### DISCUSSION TITLE
ExecuteJavascriptMiddleware cannot read the lazy load images

### DISCUSSION THREAD

**Discussion by henryonsoftware** [roach-php/core#23] [Ideas]
I'm using `ExecuteJavascriptMiddleware` in the spider but the website I want to fetch uses lazy load for their post thumbnail.
The thumbnail link will be filled to its div when we scroll to the post.
Any ideas?

**Comment by ksassnowski** [comment]
A lot implementations of lazy loaded images still store the image URL on the HTML element, just not the `src` attribute. Have you check if the div in question contains a `data-src` property or something similar that you could extract instead?
